mico-core/src/main/java/io/github/ust/mico/core/resource/ServiceResource.java

```diff
@@ -154,12 +153,13 @@
         return ResponseEntity.noContent().build();
     }
 
-    @GetMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}/{" + PATH_VARIABLE_VERSION + "}" + "/status")
-    public ResponseEntity<Resource<MicoServiceStatusResponseDTO>> getStatusOfService(@PathVariable(PATH_VARIABLE_SHORT_NAME) String shortName,
-                                                                                     @PathVariable(PATH_VARIABLE_VERSION) String version) {
-        MicoService micoService = getServiceFromMicoServiceBroker(shortName, version);
+    @GetMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}/{" + PATH_VARIABLE_VERSION + "}/{" + PATH_VARIABLE_INSTANCE_ID + "}" + "/status")
```

Comment:
This does not work well for the UI. The UI also displays the service status from the service page. This means that the UI may not know any instanceIds to use for this call. There needs to be an endpoint without instanceId that returns a list of these status DTOs

Reply:
API without an instance id is implemented now. 
Example response:
```json
{
    "_embedded": {
        "micoServiceStatusResponseDTOList": [
            {
                "shortName": "msg-validator",
                "version": "v1.0.2",
                "name": "UST-MICO/msg-validator",
                "instanceId": "msg-validator-dpgcaeaa",
                "requestedReplicas": 0,
                "availableReplicas": 0,
                "interfacesInformation": [],
                "applicationsUsingThisService": [],
                "podsInformation": [],
                "nodeMetrics": [],
                "errorMessages": [
                    {
                        "content": "No deployment of MicoService 'msg-validator' 'v1.0.2' with instance ID 'msg-validator-dpgcaeaa' is available.",
                        "type": "Error"
                    }
                ],
                "_links": {
                    "self": {
                        "href": "http://localhost:8080/services/msg-validator/v1.0.2"
                    },
                    "services": {
                        "href": "http://localhost:8080/services"
                    }
                }
            },
            {
                "shortName": "msg-validator",
                "version": "v1.0.2",
                "name": "UST-MICO/msg-validator",
                "instanceId": "msg-validator-xj64fxgg",
                "requestedReplicas": 0,
                "availableReplicas": 0,
                "interfacesInformation": [],
                "applicationsUsingThisService": [],
                "podsInformation": [],
                "nodeMetrics": [],
                "errorMessages": [
                    {
                        "content": "No deployment of MicoService 'msg-validator' 'v1.0.2' with instance ID 'msg-validator-xj64fxgg' is available.",
                        "type": "Error"
                    }
                ],
                "_links": {
                    "self": {
                        "href": "http://localhost:8080/services/msg-validator/v1.0.2"
                    },
                    "services": {
                        "href": "http://localhost:8080/services"
                    }
                }
            }
        ]
    }
}
```